src/classes/keyboard.class.js

```diff
@@ -103,6 +103,27 @@ class Keyboard {
                     return "ĉ";
                 case "C":
                     return "Ĉ";
+                // the circumflex can also be used for subscript numbers
```

Comment:
I believe you meant:
```suggestion
                // the circumflex can also be used for superscript numbers
```

Reply:
Ah yes indeed, I might have wanted to write “supscript” instead at that time